src/v2/Components/NavBar/MinimalNavBar.tsx

```diff
@@ -19,19 +21,21 @@ export const MinimalNavBar: React.FC<MinimalNavBarProps> = props => {
       width="100%"
       pt={4}
     >
-      <AppContainer>
-        <Box height={70} px={[2, 4]}>
-          <RouterLink
-            to={props.to}
-            // TODO: figure out a minimal example of the underlying cause of this error
-            // and submit an issue to TS 😓
-            // @ts-ignore
-            data-test="logoLink"
-          >
-            <ArtsyLogoBlackIcon />
-          </RouterLink>
-        </Box>
-      </AppContainer>
+      {!isEigen && (
+        <AppContainer>
+          <Box height={70} px={[2, 4]}>
+            <RouterLink
+              to={props.to}
+              // TODO: figure out a minimal example of the underlying cause of this error
+              // and submit an issue to TS 😓
+              // @ts-ignore
+              data-test="logoLink"
+            >
+              <ArtsyLogoBlackIcon />
+            </RouterLink>
+          </Box>
+        </AppContainer>
+      )}
```

Comment:
I was too slow for this, but I would prefer something like:
```
      {isEigen ? (
        null
      ) : (
        <AppContainer>
          <Box height={70} px={[2, 4]}>
            <RouterLink
              to={props.to}
              // TODO: figure out a minimal example of the underlying cause of this error
              // and submit an issue to TS 😓
              // @ts-ignore
              data-test="logoLink"
            >
              <ArtsyLogoBlackIcon />
            </RouterLink>
          </Box>
        </AppContainer>
      )}
```

Reply:
or maybe even better:

```ts
 if (isEigen) {
   return null
 }
 
 return ... 
``

Reply:
Sure, of course. It's just inline and that's why I would go for the `check ? null : rest`, but either way. Anyway, cool PR 😃!